### ISSUE TITLE
UnknownTimeZoneError: 'Timezone/Pacific'   error on starting 

### ISSUE THREAD

**Issue by rkelez** [SickRage/SickRage#1308]
Branch/Commit: master
OS: DSM 6.0 (latest official)

What you did:
- Created user "sickbeard-custom" as suggested by @neoatomic here [https://github.com/SickRage/sickrage-issues/issues/587#issuecomment-204302021](url)
- Fresh install of Python, Git, sickbeard-custom
- Repo: git://github.com/SickRage/SickRage.git

What happened:
 - SickRage fails to launch
 - SSH as root, ran start command and receive the errors below about timezones

Logs:
```
root@SynDrive:~# /var/packages/sickbeard-custom/scripts/start-stop-status start
Starting SickBeard Custom ...
Traceback (most recent call last):
  File "/usr/local/sickbeard-custom/var/SickBeard/SickBeard.py", line 94, in <module>
    import sickbeard
  File "/volume1/@appstore/sickbeard-custom/var/SickBeard/sickbeard/__init__.py", line 38, in <module>
    from sickbeard.config import CheckSection, check_setting_int, check_setting_str, check_setting_float, ConfigMigrator
  File "/volume1/@appstore/sickbeard-custom/var/SickBeard/sickbeard/config.py", line 29, in <module>
    from sickbeard import naming
  File "/volume1/@appstore/sickbeard-custom/var/SickBeard/sickbeard/naming.py", line 27, in <module>
    from sickbeard import tv
  File "/volume1/@appstore/sickbeard-custom/var/SickBeard/sickbeard/tv.py", line 51, in <module>
    from sickbeard import network_timezones
  File "/volume1/@appstore/sickbeard-custom/var/SickBeard/sickbeard/network_timezones.py", line 34, in <module>
    sb_timezone = get_localzone()
  File "/usr/local/sickbeard-custom/var/SickBeard/lib/tzlocal/unix.py", line 122, in get_localzone
    _cache_tz = _get_localzone()
  File "/usr/local/sickbeard-custom/var/SickBeard/lib/tzlocal/unix.py", line 91, in _get_localzone
    return pytz.timezone(etctz.replace(' ', '_'))
  File "/usr/local/sickbeard-custom/var/SickBeard/lib/pytz/__init__.py", line 180, in timezone
    raise UnknownTimeZoneError(zone)
pytz.exceptions.UnknownTimeZoneError: 'Timezone/Pacific'

```

I'm fairly new, if I've missed something I'm happy to provide any further assistance. I don't know how related the issues are but everything worked fine before the DSM 6.0 upgrade.

**Comment by SickRage2** [comment]
Thanks for the issue report! Before a real human comes by, please make sure your report has all the below criteria checked

- [ ] Include basic information: Branch/Commit, OS, What you did, What happened, What you expected
- [ ] Enable debug logging (be sure to disable after the bug is fixed)
- [ ] Post debug logs, either inline (for smaller logs) or using gist

Please make sure you also read [how to create an issue](https://github.com/SickRage/sickrage-issues/blob/master/README.md) and followed all of the steps. 

The title should describe your issue. Having "SR not working" or "I get this bug" for 100 issues, isn't really helpful. We will close issues if there isn't enough information.

Sometimes the devs may seem like grunts and respond with short answers. This isn't (always) because the dev hates you, but because he's on mobile or busy fixing bugs. If something isn't clear, please let us know, and this bot may get updated to automatically answer you.

Thanks!

**Comment by neoatomic** [comment]
That is not the proper way to start the Synology package over ssh, also the Synocommunity installer/package is not fully compatible with DSM6 at this point.
See this issue for more info. : https://github.com/SickRage/sickrage-issues/issues/587

**Comment by andyblac** [comment]
i have no issues with DSM 6.0, we where just trying to give you as much info as possible.

**Comment by neoatomic** [comment]
@andyblac We are not maintaining the Synology installer our self's. That is done by the Synocommunity folks.
So we have no influence on that. (or can repair it) Also the error above is probbebly related to permissions, caused by running with sudo/wrong user over ssh etc.

Feel free to discuss it, but please dont open new issues as there is already an open issue for the DSM6 issues. Also the workarounds are just that, we cant "officially" support them here. 
In short, we simply have to wait till Synocommunity fixes/update's there installer package.

**Comment by andyblac** [comment]
as i said i do not believe this is an DSM issue as it run's perfectly fine on my DS3615XS, (my own package), you you add some debug for TZ detection and where it is trying to read from etc ?

**Comment by neoatomic** [comment]
@miigotu I leave this one for you to review and shad your light on.
(But my gut tells me its a permissions issue, however cant exclude another cause also...)

**Comment by andyblac** [comment]
on dms 6 it seem to be in
```
/usr/share/zoneinfo/Timezone
```
![screen shot 2016-04-01 at 15 14 36](https://cloud.githubusercontent.com/assets/77936/14209549/7b6ce900-f81c-11e5-90f5-c14b62e80ed3.png)

**Comment by miigotu** [comment]
Update and try again please.

**Comment by miigotu** [comment]
Its possible synology fubar'd this all up.

It should be like:
`/usr/share/zoneinfo/US/Eastern`
or
`/usr/share/zoneinfo/America/Eastern`

Need to know if https://github.com/SickRage/SickRage/commit/c7f111c0213a4eaea4f274575a468c83ba2fde2d fixes it or not.

**Comment by andyblac** [comment]
i'm in the UK so use GMT timezone, maybe why mine is working? anyway here is a another screen shot if it helps
![screen shot 2016-04-01 at 18 33 34](https://cloud.githubusercontent.com/assets/77936/14215031/566c72b2-f838-11e5-9dae-644906c62968.png)

**Comment by miigotu** [comment]
`ls -l /usr/share/zoneinfo/Timezone/Pacific`

Does that point to `/usr/share/zoneinfo/US/Pacific`?

**Comment by rkelez** [comment]
Hi, thanks for the replies. As @andyblac noted he was helping me troubleshoot on this last night. He's certainly more knowledgeable than I am.

If there's anything I can set permissions for i'm happy to @neoatomic. This is essentially a fresh install though. But if i have to wait for synocommunity i'll just have to wait. Wanted to make sure i did everything i could.

@miigotu I just ran that command in terminal and get the following

'''''''''''''''
root@SynDrive:~# ls -l /usr/share/zoneinfo/Timezone/Pacific
-rw-r--r-- 3 root root 2901 Mar 15 11:33 /usr/share/zoneinfo/Timezone/Pacific
root@SynDrive:~# 
''''''''''''''''

Happy to run anything else if it's useful. 

Thank you all!

**Comment by miigotu** [comment]
Just try reinstalling it, I pushed a fix for it.

**Comment by rkelez** [comment]
@miigotu OMG. That appears to have worked!!!!!

In case it's helpful, these are the steps I've used now.

1. Uninstall Python, Git, Sickbeard-custom
2. Install Python, Git
3. Create a "sickbeard-custom" DSM user. Add to the Admin group
4. Install sickbeard-custom. Repo: https://github.com/SickRage/SickRage.git

That's on a DS215j, that was upgraded from DSM 5.2 -> DSM 6.0 that was previously not working. 


Thank you so much. I'm restoring my db now and hoping everything goes smooth. Thanks so much to all of you :+1:

**Comment by arijbrown** [comment]
@miigotu - any chance you can give an overview of what you fixed?  I'm seeing other apps (specifically headphones) that seem to have the same issue on DSM 6.x - since they don't seem to have fixed it I'd love to know what's up.

**Comment by miigotu** [comment]
Its a bug in pytz, I just switched back to dateutil
